code/modules/capture_the_flag/ctf_game.dm

```diff
@@ -256,9 +256,17 @@
 		anchored = TRUE // Avoid directly assigning to anchored and prefer to use set_anchored() on normal circumstances.
 		return
 	//passing means the user picked up the flag so we can now apply this
+	to_chat(user, span_userdanger("Take \the [initial(name)] to your team's controller!"))
 	user.set_anchored(TRUE)
 	user.status_flags &= ~CANPUSH
 
+/obj/item/ctf_flag/attackby(obj/item/item, mob/user, params)
+	if(istype(item, /obj/item/ctf_flag))
+		var/obj/item/ctf_flag/flag = item
+		if(flag.team != team)
+			to_chat(user, span_userdanger("Take \the [initial(flag.name)] to your team's controller!"))
+			user.playsound_local(get_turf(user), 'sound/machines/buzz-sigh.ogg', 100, vary = FALSE, use_reverb = FALSE)
```

Comment:
```suggestion
	if(!istype(item, /obj/item/ctf_flag))
		return ..()
		
	var/obj/item/ctf_flag/flag = item
	if(flag.team != team)
		to_chat(user, span_userdanger("Take \the [initial(flag.name)] to your team's controller!"))
		user.playsound_local(get_turf(user), 'sound/machines/buzz-sigh.ogg', 100, vary = FALSE, use_reverb = FALSE)
```

yeah i don't see why we should be fully overwriting parent anyways, and also early returns make the indentation saner.